### PR TITLE
Pushing FAQ to dev build

### PR DESCRIPTION
**Pull request by ZwarA01**
Finished FAQ page, able to access the FAQ through the settings cog

### CODE REVIEW

**Review by majedau — APPROVED** [review]
Looks good to me! Good job on the UI. However, I would recommend next time add more color so that it matches our scheme with the overview pages. Such as in line 44, you could change the bg tag to another color to stand out more.

**Review by Greysushii — APPROVED** [review]
Looks good, I agree with Majeda. Also maybe we can implement this as a component to help cut the code length down? Just a thought!

File: pages/questions.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            <div className="p-4 bg-white rounded-lg border border-gray-200 mb-6">
+              <div className="justify-between items-center mb-3 sm:flex">
+                  <a href="#" className="text-sm font-semibold text-gray-900">Lorem ipsum dolor sit amet?</a>
+              </div>
+              <div class="p-3 text-xs italic font-normal text-gray-500 bg-[#f9f9f9] rounded-lg border border-gray-200">Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</div>
+            </div>
+
+            <div className="p-4 bg-white rounded-lg border border-gray-200 mb-6">
+              <div className="justify-between items-center mb-3 sm:flex">
+                  <a href="#" className="text-sm font-semibold text-gray-900">Consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore aliqua?</a>
```
Comment (PattyCoding123) [inline pages/questions.js:56]: Make sure to include other types of frequently asked questions and responses to the page later on.

File: pages/questions.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+              </div>
+              <div className="p-3 text-xs italic font-normal text-gray-500 bg-[#f9f9f9] rounded-lg border border-gray-200">Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</div>
+            </div>
+
+            <div className="p-4 bg-white rounded-lg border border-gray-200 mb-6">
+              <div className="justify-between items-center mb-3 sm:flex">
+                  <a href="#" className="text-sm font-semibold text-gray-900">Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur?</a>
```
Comment (PattyCoding123) [inline pages/questions.js:63]: Same here, add more information/

**Review by PattyCoding123 — APPROVED** [review]

**Pull request merged**